code/analytics/src/phone/java/com/adobe/marketing/mobile/Analytics.java

```diff
@@ -243,11 +245,12 @@ public void fail(AdobeError adobeError) {
     }
 
     /**
-     * Retrieves the visitor identifier
+     * Sets the visitor identifier. Clears the visitor identifier when passed null or an empty
+     * string.
      *
      * @param visitorID {@code String} new value for visitor identifier
      */
-    public static void setVisitorIdentifier(@NonNull final String visitorID) {
+    public static void setVisitorIdentifier(@Nullable final String visitorID) {
```

Comment:
Public API change. Previous Android 1.x allowed null values to clear the VID so we want to keep that behavior in 2.x.

Reply:
👍  make sure Documentation/api-reference is in sync

Reply:
Thanks. I've updated the api-reference documentation.

Reply:
and kotlin String?